specs/altair/sync-protocol.md

```diff
@@ -94,9 +86,10 @@ class LightClientUpdate(Container):
 ### `LightClientStore`
 
 ```python
-class LightClientStore(Container):
+@dataclass
+class LightClientStore(object):
     snapshot: LightClientSnapshot
-    valid_updates: List[LightClientUpdate, MAX_VALID_LIGHT_CLIENT_UPDATES]
+    valid_updates: Set[LightClientUpdate]
```

Comment:
rationale for this change:

decided to go w/ `Set` here as `List` has a local definition as the `ssz` type and python's `typing.List` that we want here would collide by name...

rather than have to juggle a type alias `PythonList` that would need to be defined for spec consumers, or use the fully qualified name `typing.List` (that would not directly work w/ our spec builder machinery), we can just move to a `Set` datatype -- which satisfies the needs of `valid_updates` just the same